maccabipediabot/common.py

```diff
@@ -71,18 +83,25 @@ def format_season_id(season):
     :rtype: str
     """
     if season in ["נוכחית", "הנוכחית", "אחרונה", "האחרונה"]:
-        return _CURRENT_SEASON
+        return get_current_season()
 
     # Avoid non numbers cases (for the two possible separators) and non valid formats
     if not season.split('-')[0].split('/')[0].isdigit():
         return season  # Nothing to do with this format
-    if len(season) == 7 or len(season) == 5:
-        if not ('-' in season or '/' in season):
+
+    season = season.replace('-', '/')  # Replacing '-' with '/' to get correct format
+
+    if len(season) == 9 or len(season) == 7 or len(season) == 5:
+        if not ('/' in season):
             return season  # Nothing to do with this format
 
-    if len(season) == 7:  # Normal format --> 1995/96 or 1995-96
-        return season.replace("-", "/")
-    elif len(season) == 5:  # Short format --> 95/96 or 95-96
+    if len(season) == 9:  # Long format --> 1995/1996
+        season = season.split('/')
```

Comment:
Use different variable, not season.

user_seasons = season.split("/")

and you forgot to return